Makefile

```diff
@@ -75,7 +75,7 @@ start-mongo:
      -v $(shell pwd)/test/fixtures/mongo_keyfile:/data/mgkey \
 	mongo:4.4.1 mongod --port=17017 --replSet=skynet --keyFile=/data/mgkey 1>/dev/null 2>&1
 	# wait for mongo to start before we try to configure it
-	sleep 3
+	sleep 4
```

Comment:
Is there no better / more deterministic way to achieve this? e.g. by waiting in a loop for 60 seconds and trying to talk to mongo every second or so?

Reply:
I was putting this off because I knew it would take some time but... yeah, it had to be done. :)